backend/routes.js

```diff
@@ -69,6 +92,17 @@ route.POST('/event', {
         const parsedManifest = JSON.parse(manifest)
         const { name } = JSON.parse(parsedManifest.body)
         if (name !== 'gi.contracts/identity') return Boom.unauthorized('This contract type requires ownership information', 'shelter')
+        // rate limit signups in production
+        if (process.env.NODE_ENV === 'production') {
```

Comment:
I think that it should be possible to disable this functionality altogether to allow for this to be done at a lower level (yes, it needs to know about contract types, but it's possible to do this with things like OpenResty or commercial WAF offerings such as Cloudflare)